### PR TITLE
Adds Google Maps card to the weather page

### PR DESCRIPTION
**Pull request by cjoshea9**
Adds styled Google Maps widget to the weather page to show the weather at the various Google office locations around the globe.

### CODE REVIEW

File: portfolio/src/main/webapp/weather.js [hunk 1]
```diff
@@ -50,14 +50,74 @@ function updateWeather(lat, lon, city) {
...
-    }
-
     weather();
   });
+}
 
+/**
+* Rounds input to one decimal
+*/
+  function round(num) {
```
Comment (g-harel) [inline portfolio/src/main/webapp/weather.js:60]: Nit: Indentation

File: portfolio/src/main/webapp/weather.js [hunk 2]
```diff
@@ -50,14 +50,74 @@ function updateWeather(lat, lon, city) {
...
+  return Math.round(num*10)/10;
 }
+
+//Loads Google Charts API with API key
+const keyFile = new XMLHttpRequest();
+let mapsApiKey = '';
+keyFile.open("GET", "key.txt", false);
```
Comment (g-harel) [inline portfolio/src/main/webapp/weather.js:67]: Seems like it's accessible to anyone on the internet anyways. If that's the case, there isn't much of a benefit to the indirection.

Comment (cjoshea9) [reply portfolio/src/main/webapp/weather.js:67]: I got an email from GitHub saying that they had found a Google API key on my repo and that I should hide it. I don't know if it was necessary but better safe than sorry I guess. @g-harel

File: portfolio/src/main/webapp/weather.js (hunk 1 above)
Comment (cjoshea9) [reply portfolio/src/main/webapp/weather.js]: Fixed.

File: portfolio/src/main/webapp/weather.js (hunk 2 above)
Comment (g-harel) [reply portfolio/src/main/webapp/weather.js:67]: Up to you! You could use jQuery for this request too, would make it much simpler.

**Pull request merged**